devops/ansible/roles/php70/vars/main.yml

```diff
@@ -9,7 +9,7 @@ debug_mode: false
 # allow_url_fopen
 #   Default Value: On
 php_allow_url_fopen: "Off"
-php_disable_functions: "exec,passthru,shell_exec,system,proc_open,popen"
+php_disable_functions: "exec,passthru,shell_exec,system,popen"
```

Comment:
There was an error when running a wp cli command. So I removed proc_open and it worked.  I think the error only pops up for certain commands though. I think I can add proc_open back to that script.